cmd/dev/headers/copyright.go

```diff
@@ -67,10 +70,19 @@ func AddHeaders(dir string, year int, template string, exclude []string) error {
 		if pathContainsFolders(path, exclude) {
 			return nil
 		}
-		contentNoHeader, err := comments.FileContentWithoutHeader(path, HEADER_TOKEN)
+		format, ok := comments.GetFormat(path)
+		if !ok {
+			return nil
+		}
+		content, err := comments.FileContent(path)
 		if err != nil {
 			return err
 		}
+		header, contentNoHeader := format.SplitHeaderFromContent(content, HEADER_TOKEN)
```

Comment:
I would be more comfortable if this would test for:

```
// Copyright © \d{4} Ory Corp
```

Reply:
ok